x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -513,10 +513,7 @@ export const UserProfile: FunctionComponent<UserProfileProps> = ({ user, data })
           <EuiPageTemplate
             style={{ backgroundColor: euiTheme.colors.emptyShade }}
```

Comment:
Let's go ahead and remove this `style` for the background color as well, so that the default light gray background will show through on the header and provide some contrast between header and body.

```suggestion
```

Reply:
Good call - Removed in latest commit!